Model/Player.py

```diff
@@ -0,0 +1,26 @@
+from Server.Database.DBMethods import DBMethods
+
+
+class Player:
+    def __init__(self, player_id, username, password):
+        self.id = player_id
+        self.username = username
+        self.password = password
+
+    def to_dict(self):
+        return self.__dict__
+
+    @staticmethod
+    def create_object_from_details_in_db(player_username):
+        player_as_dict = DBMethods.get_player_details_from_db(player_username)
```

Comment:
DBMethods.get_player_details_from_db -> the class name is DBMethods which means it work with the DB so you can make the function name shorter: 
get_player_details_from_db -> fetch_player / get_player